### ISSUE TITLE
Exception if meta contains non-JSON-serializable data

### ISSUE THREAD

**Issue by wolever** [logdna/python#56]
To reproduce:

```
log.info("foo", { "meta": object() })
```

And notice that this line will fail:

```
message['meta'] = json.dumps(opts['meta']) 
```

I believe:

1. This should be a logged failure (instead of crashing the app), and
2. The library should allow a custom JSON serializer to be provided (which can accommodate  JSON serialization for types like `datetime`)

**Comment by dhudsmith** [comment]
I also have this issue.

**Comment by smusali** [comment]
@wolever, sorry for the late response!

#70 introduces the meta-specific changes which have already been released in [`v1.17.0`](https://github.com/logdna/python/releases/tag/v1.17.0).

Please, let us know if you have any additional questions!

Thanks for using LogDNA!

_Happy Logging!_
